### ISSUE TITLE
salt-cloud (EC2): Salt-Minion-2014.7.2-AMD64-Setup.exe hangs forever

### ISSUE THREAD

**Issue by dhs-rec** [saltstack/salt#21258]
Hi,

because of the problem with Salt-Minion-2014.7.1-AMD64-Setup.exe during a salt-cloud run (see #21256), I gave 2014.7.2 a try, but this time the installer just hangs:
```
[INFO    ] Running command under pid 18798: 'smbclient -U Administrator%secret_passwd //11.22.33.44/c$ -c "cd salttemp; prompt; lcd /root; mput Salt-Minion-2014.7.2-AMD64-Setup.exe; exit;"'
Domain=[IP-ABCDEF] OS=[Windows Server 2008 R2 Datacenter 7601 Service Pack 1] Server=[Windows Server 2008 R2 Datacenter 6.1]
putting file Salt-Minion-2014.7.2-AMD64-Setup.exe as \salttemp\Salt-Minion-2014.7.2-AMD64-Setup.exe (1524.1 kb/s) (average 1524.1 kb/s)
[INFO    ] Running command under pid 18804: 'winexe -U Administrator%secret_passwd //11.22.33.44 "c:\\salttemp\\Salt-Minion-2014.7.2-AMD64-Setup.exe /S /master=ip-10-11-12-13.region.compute.internal /minion-name=salt-cloud-test"'
```

Bye...

Dirk

**Comment by dhs-rec** [comment]
Anything I can do to help resolving this issue?

**Comment by freimer** [comment]
It's probably a bad password quote. See #21188, but I'm not sure that is the proper way to handle it.  Works for me, and if you want, you can put the updated .py file in _modules for testing.

**Comment by dhs-rec** [comment]
Yes, it fixes this issue, but now I also see #21256 with installer version 2014.7.2.

BTW: Installed salt-common and salt-cloud packages are 2014.7.1+ds-3trusty1, since that's the most recent version in the Ubuntu PPA.

**Comment by jfindlay** [comment]
@dhs-rec, do you consider #21188 to fix this issue?  If so, can it be closed?  Thanks.

**Comment by dhs-rec** [comment]
Yes, this issue seems to be fixed with #21188. Closing...
